swagger.yaml

```diff
@@ -403,6 +413,29 @@ parameters:
     type: string
     required: true
     description: '"Bearer [token]" where [token] is your OAuth2 access token'
+  addressType:
+    name: addressType
+    in: query
+    description: |
+      Filter by phone type. Common phone types and their descriptions:
+      - "CM" - Current Mailing
+      - "PA" - Student Alternate Contact
+      - "DM" - Department (US Mail)
+      - "RK" - Financial Aid Address
+      - "RH" - Residence Hall
+    required: false
```

Comment:
Although it's not required for calling the SP, I think we probably don't want out client to insert a phone without `addressType`. What do you think?

Reply:
True but these are for the get endpoints. I could be wrong but I don't think these parameters are referenced in the post body definitions.

Reply:
@Ruefa oh yeah you are right. My brain doesn't work well today 😩 Feel free to ignore my comment here.